src/helm/benchmark/static/schema_vlm.yaml

```diff
@@ -416,6 +418,40 @@ run_groups:
       who: Human experts
       when: "2023"
       language: English, Chinese, Japanese, etc.
+  
+  - name: multipanelvqa
+    display_name: MultipanelVQA
+    description: Question about real-world or synthetic multipanel images for evaluating multi-panel image reasoning ability
+    metric_groups:
+      - accuracy
+      - bias
```

Comment:
For both of these scenarios, could you just have the following in `metric_groups`:

      - accuracy
      - efficiency
      - general_information

Reply:
Sure.